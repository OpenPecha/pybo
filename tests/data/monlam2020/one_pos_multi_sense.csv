edited,favorites,_id,word,result
-,-,2,ཀ་ཀ་, མིང་ཚིག 1. བྱ་སྐྱ་ཀ 2. བྱ་སྐྱ་ཀའི་སྐད། 3. ལུག་གི་ཐེ་གེའམ་ཨ་ཅུག 4. ༡ཡུལ་སྐད།  བྱིས་པ་ཆུང་ངུའི་གྱོན་པ་ལ་ཀ་ཀ་ཟེར། དཔེ་རིས་ལ་གཟིགས།
