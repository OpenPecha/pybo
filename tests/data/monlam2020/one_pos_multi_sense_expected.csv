ID,Form,Lemma,MonPOS,MonFeature,MonTag,POS,Feature,Morph,SenseTag,Definition,Example
1,ཀ་ཀ་,ཀ་ཀ་1, མིང་ཚིག,   ,,,,,བྱ་,བྱ་སྐྱ་ཀ,
,,ཀ་ཀ་2,,,,,,,བྱ་,བྱ་སྐྱ་ཀའི་སྐད།,
,,ཀ་ཀ་3,,,,,,,ལུག་,ལུག་གི་ཐེ་གེའམ་ཨ་ཅུག ,
,,ཀ་ཀ་4,,,༡ཡུལ་སྐད།,,,,བྱིས་པ་,བྱིས་པ་ཆུང་ངུའི་གྱོན་པ་ལ་ཀ་ཀ་ཟེར། དཔེ་རིས་ལ་གཟིགས།,
